### ISSUE TITLE
ROI module not working

### ISSUE THREAD

**Issue by onurserce** [sgoldenlab/simba#13]
**Describe the bug**
ROI module is not working. After selecting the ROI's, the output CSV file appears to be empty.

**To Reproduce**
After loading the project
1. Go to 'ROI' tab
2. Define ROI's and click 'Show Shape Definitions table'
3. Enter shape names, leave dimensions 0, click on Set Shape Definitions
4. There's only one video, click on draw and draw shapes according to instructions
5. Click on 'Analyse ROI Data'

**Expected behavior**
A CSV file with analyzed ROI's

**Screenshots**
![bug1](https://user-images.githubusercontent.com/31575364/77079048-b7389a00-69f7-11ea-9b53-2787a463fd8d.PNG)
![bug2](https://user-images.githubusercontent.com/31575364/77079052-b869c700-69f7-11ea-827d-7a2136e6e60a.PNG)
![bug3](https://user-images.githubusercontent.com/31575364/77079054-b9025d80-69f7-11ea-94eb-69df0206bb55.PNG)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

**Additional context**
1. Outlier correction step was done successfully and non-empty .csv files are available at project_folder\csv\outlier_corrected_movement_location and project_folder\csv\outlier_corrected_movement
2. Video name in the file project_folder\logs\video_info.csv was changed due to incompatibility of Simba with Deeplabcut while importing videos into Simba ( #8 )
3. All .csv files highlighted in the screenshot are empty.

**Comment by sgoldenlab** [comment]
Thanks @onurserce - you did catch a bug with the file import, some CSV tracking file names were not changed  appropriately when imported into SimBA - primarily as DeepLabCut has changed it output file names to contain "DLC" rather than "DeepCut" - I will have an updated version online shortly and get back to you.  

In the meantime, is it right that you have a space in your file name ("Test 2DLC...") ? SimBA and python are not very good at handling spaces. Try removing it before you importing it (e.g., import a video named Test.mp4, and a tracking file named Test.cvs) and let me know how it goes.

**Comment by sgoldenlab** [comment]
I have fixed the bug - could you download SimBA again, start a new project, and give it a go without spaces in the filenames, and let me know how it works?

**Comment by sgoldenlab** [comment]
Confirmed fixed by another user. Thanks!

**Comment by onurserce** [comment]
@sgoldenlab Indeed I had a lot of trouble due to that space in the filename and I had to change the project_folder\videos\Test 2 to project_folder\videos\Test and that fixed the problem encountered during the frame extraction step.

However, even though I reinstalled SimBA. ROI module still seems to be not working for me, I just have two more empty CSV files under logs folder. Should I try something else?
